runtime/compiler/optimizer/IdiomRecognition.cpp

```diff
@@ -1048,7 +1048,7 @@ TR_CISCGraph::makePreparedCISCGraphs(TR::Compilation *c)
    // FIXME: We need getSupportsCountDecimalDigit() like interface
    // this idiom is only enabled on 390 for the moment
    //
-   bool genDecimal = TR::Compiler->target.cpu.isZ();
+   bool genDecimal = TR::Compiler->target.cpu.isZ() && !c->isOutOfProcessCompilation();
```

Comment:
Please add a short comment specifying why this cannot be done for JITServer

Reply:
a2363071256e93fd7372c45cc7725394d547c194